### PR TITLE
Tests with empty names are displayed with a defualt name

### PR DESCRIPTION
**Pull request by meoke**

### CODE REVIEW

File: src/Widgets.jl [hunk 1]
```diff
@@ -73,9 +73,9 @@ function create_details_textbox!(frame::Tk_Frame, test_details::AbstractString)
   pack(details_box)
 end
 
-function get_node_label!(frame::Tk_Frame,tests_structure::Vector{TestStructureNode}, test_node::FactNode,nesting_level::Int)
+function get_node_label!(frame::Tk_Frame, tests_structure::Vector{TestStructureNode}, test_node::FactNode,nesting_level::Int)
   frame_for_tests = get_frame_for_tests(frame)
-  button_text = name(test_node)
+  button_text = name(test_node) != "" ? name(test_node) : "Test"
```
Comment (gdziadkiewicz) [inline src/Widgets.jl]: Wouldn't something like "Unnamed Test" be better?

File: src/Widgets.jl [hunk 2]
```diff
@@ -91,9 +91,9 @@ function create_line_number_button!(node_label::Tk_Label, test_node::FactNode)
   bind(line_number_button, "<Button-1>", _ -> line_number_button_callback(test_node, get_file_name()))
 end
 
-function get_node_label!(frame::Tk_Frame,tests_structure::Vector{TestStructureNode},test_node::TestStructureNode, nesting_level::Int)
+function get_node_label!(frame::Tk_Frame, tests_structure::Vector{TestStructureNode},test_node::TestStructureNode, nesting_level::Int)
   frame_for_tests = get_frame_for_tests(frame)
-  button_text = name(test_node)
+  button_text = name(test_node) != "" ? name(test_node) : "Group of tests"
```
Comment (gdziadkiewicz) [inline src/Widgets.jl]: As above

File: src/Widgets.jl (hunk 1 above)
Comment (meoke) [inline src/Widgets.jl]: You're right, I'll change it.

File: src/Widgets.jl (hunk 2 above)
Comment (meoke) [inline src/Widgets.jl]: As above.

**Pull request merged**